lib/reek/tree_dresser.rb

```diff
@@ -10,6 +10,47 @@ def initialize(klass_map = AST::ASTNodeClassMap.new)
       @klass_map = klass_map
     end
 
+    # Given this sexp (Parser::AST::Node):
```

Comment:
For this comment, I'm missing the big picture a bit here. How about starting with something like:

> Recursively enhance an AST with type-dependent mixins, and comments.